synfig-core/src/synfig/rendering/software/task/taskpaintpixelsw.cpp

```diff
@@ -122,3 +122,71 @@ synfig::rendering::TaskPaintPixelSW::run_task() const
 
 	return true;
 }
+
+bool
+synfig::rendering::TaskFilterPixelSW::run_task() const
+{
+	const TaskPixelProcessorBase* task = dynamic_cast<const TaskPixelProcessorBase*>(this);
+	if (!task) {
+		const Task* task = dynamic_cast<const Task*>(this);
+		if (task)
+			synfig::error(_("Internal error: Cobra task %s isn't a TaskPixelProcessorBase"), task->get_token()->name.c_str());
+		else
+			synfig::error(_("Internal error: this TaskPaintPixelSW isn't even a Task"));
```

Comment:
Is this correct? 
I mean `TaskPaintPixelSW`? Looks like should be `TaskFilterPixelSW`?